### PR TITLE
Content terms and risk

### PR DESCRIPTION
**Pull request by AlexDuchnowski**
### Pull Request
---

#### PR Summary
The addition of the Terms of Service for the Oscar details page, the addition of data risk levels for 'Computing' Services, and new icons for the About page

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [x] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [ ] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

File: content/english/services/_index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ hero:
     cases involve a researcher using grant funds to support one of our data scientists
     or research software engineers. '
   cfa:
-    text: See our rates
+    text: See our Rates
```
Comment (mcmcgrath13) [inline content/english/services/_index.md]: why capitalize rates?

File: content/english/services/infrastructure/hpc-condo.md [hunk 2]
```diff
@@ -22,3 +25,26 @@ CCV standardizes processor and memory configurations for a period of roughly six
 
 # Purchasing
 Pricing is dependent on the size of the condo, and will be provided upon request. Please contact CCV at [support@ccv.brown.edu](mailto:support@ccv.brown.edu).
+
```
Comment (mcmcgrath13) [inline content/english/services/infrastructure/hpc-condo.md]: remove empty lines

File: themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mcmcgrath13) [inline themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json]: are these files added by your IDE? If so, add them to the gitignore and delete them from the PR

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]
🚀 I still need to look at the changes in the browser, but my computer decided I needed to reinstall a bunch of software first...

**Comment by fernandogelin** [comment]
Link to the deployment of this PR: https://review-app-content-term-k09f0u.herokuapp.com

File: content/english/services/_index.md (hunk 1 above)
Comment (AlexDuchnowski) [reply content/english/services/_index.md]: Not sure... I'll change it back.

File: themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json (hunk 3 above)
Comment (AlexDuchnowski) [reply themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json]: Yep, Visual Studios added those. How do I add them to the gitignore and delete them from the PR?

**Comment by mcmcgrath13** [comment]
Some more thoughts from reviewing in the browser:

* Why only have the risk classification on compute - isn't that more of a storage thing?
  * with the risk classification on all of those, the styling feels a bit agressive -  maybe make it a bit more subtle/friendly?
* In the terms of service the ccv rates link is hard coded, should be relative
* I prefer the crow logo for the mission in about

File: themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json (hunk 3 above)
Comment (mcmcgrath13) [reply themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json]: add a glob that matches them in `.gitignore` (https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/)

and then https://devconnected.com/how-to-delete-file-on-git/#:~:text=The%20easiest%20way%20to%20delete,the%20file%20to%20be%20deleted.&text=Note%20that%20by%20using%20the,be%20deleted%20from%20the%20filesystem. for removing the files

Comment (AlexDuchnowski) [reply themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json]: I just tried adding the glob, removing the files, committing, and then pushing, but I can't tell if it worked. Did it?

Comment (AlexDuchnowski) [reply themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json]: I tried one more time, and I think it has worked. Please let me know if it hasn't. Thanks!

Comment (mcmcgrath13) [reply themes/gb-theme/layouts/about/.vs/VSWorkspaceState.json]: looks good to me!

**Comment by AlexDuchnowski** [comment]
@mcmcgrath13 

> Some more thoughts from reviewing in the browser:
> 
> * Why only have the risk classification on compute - isn't that more of a storage thing?
>   
>   * with the risk classification on all of those, the styling feels a bit agressive -  maybe make it a bit more subtle/friendly?
> * In the terms of service the ccv rates link is hard coded, should be relative
> * I prefer the crow logo for the mission in about

I've changed the rates link.

I'd like to try changing the color scheme for the data risk classification labels/buttons, but I'm having trouble finding the right file to edit (I'll ask around on Slack).

I'm happy to change the mission icon back to the crow, but I was thinking that it would make more sense to have an icon that is somehow related to the concept of a mission statement.

Here are a few alternatives:
- [Road](https://fontawesome.com/icons/road?style=solid)
- [Globe-americas](https://fontawesome.com/icons/globe-americas?style=solid)
- [Globe](https://fontawesome.com/icons/globe?style=solid)
- [Location-arrow](https://fontawesome.com/icons/location-arrow?style=solid)

If you don't think any of these are better, I'll change it back right away. Thanks for all the feedback!

**Comment by AlexDuchnowski** [comment]
The pull request is ready for re-review now.

**Review by mcmcgrath13 — APPROVED** [review]
🚀

**Comment by AlexDuchnowski** [comment]
@paulstey @mttunca - Happy New Year! Is there anything you would like me to change here, or can this request be merged now?

Thanks in advance,

Alex

**Review by paulstey — APPROVED** [review]

**Pull request merged**